src/form/Form.js

```diff
@@ -19,15 +19,18 @@
             .columns([{
                 id: "subject",
                 label: "Subject",
-                type: "string"  //  If ommitted should default to string  ---
+                type: "string",  //  If ommitted should default to string  ---
+                validate: "[A-Za-z]"
```

Comment:
yea that's becuase regex still matched it .. gotta use something like ^[A-z]+$ ... this part isnt final and not even sure if we will include ... J has a refactor as well I need to look at